.github/workflows/test_pr_changes.yml

```
@@ -45,6 +47,12 @@ jobs:
           filters: |
             wire-ios:
               - 'wire-ios/**'
+            wire-reusable-ui-components:
+              - 'WireUI/Sources/WireReusableUIComponents/**'
+              - 'WireUI/Tests/WireReusableUIComponentsTests/**'
+            wire-design:
+              - 'WireUI/Sources/WireDesign/**'
+              - 'WireUI/Tests/WireDesignTests/**'
```

I think we should actually stop testing individual targets only.
For example: changes in WireDesign directly affect Wire-iOS, so I might change these lines into:
```suggestion
            wire-ios:
              - 'wire-ios/**'
              - 'WireUI/**'
```
Please share your opinions.

---

This (test ui when WireUI changes) makes sense to me. I'm not sure though how expensive it will be to test every single target on every change. I guess once we have fully migrated to SPM it would be easier to do selective testing?